Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
This `stdout` leaking is caused by losing the scope. You are actually calling `Kernel.system` instead of `Formula.system`.

The right solution is leaving `include Language::JS` as it is. We did the same thing in Haskell(e.g. `pandoc.rb`).